dashboard/src/routes/hardware/$hardwareId/route.tsx

```diff
@@ -6,6 +6,7 @@ import { zPossibleValidator, zTableFilterInfo } from '@/types/tree/TreeDetails';
 
 const hardwareDetailsSearchSchema = z.object({
   currentPageTab: zPossibleValidator,
+  treeFilter: z.array(z.number().int()).optional(),
```

Comment:
nit: `treeIndexesList` or `treeSelecteds` are more suggestive

Reply:
I'll use `treeIndexes` so to not make it too long or too illogical. I'm also renaming it for that name in hardwareDetails already anyway.